lib/omnibus/software.rb

```diff
@@ -767,6 +746,19 @@ def overridden?
     # @!endgroup
     # --------------------------------------------------
 
+    #
+    # Path to where any source is extracted to.
+    #
+    # Files in a source directory are staged underneath here. Files from
+    # a url are fetched and extracted here. Look outside this directory
+    # at your own peril.
+    #
+    # @return [String] the full absolute path to the package root directory.
+    #
+    def package_dir
```

Comment:
Actually seeing as how package_dir is used in the packages, I would be a terrible person to disambiguate project_dir with "the directory this software def is built in" only to then confuse it with the packager output.  I've updated the PR by calling it fetch_dir instead.  It might be a bit weird inside the git_fetcher (since its "project_dir" might be the "fetch_dir" where neither project nor fetch have anything to do with git), I think in isolation, each file makes fairly good sense with that name.  Once we have the options/ability to lose the back-compat features, it should be a whole lot less weird.